torchvision/models/detection/backbone_utils.py

```diff
@@ -87,7 +87,7 @@ def resnet_fpn_backbone(
         norm_layer=norm_layer)
 
     # select layers that wont be frozen
-    assert trainable_layers <= 5 and trainable_layers >= 0
+    assert 5 >= trainable_layers >= 0
```

Comment:
Haha, maybe `0 <= trainable_layers <=5` is more readable?

Reply:
Great feedback for my IDE. Let me change this. :D